app/posts/publish-teacher-training-courses/2021-06-03-placement-schools-and-training-centres.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Placement schools and training centres
+description: Asking providers to give more detailed location information
+date: 2021-06-14
+---
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+
+## The location problem
+
+Currently, Publish uses a model inherited from UCAS. This allows courses to show 2 types of location:
+
+* the training provider’s contact address
+their training locations – places a candidate
+* can apply to, but not necessarily where they will spend the majority, if any, of their time.
+
+Providers can add up to 37 addresses. Some set themselves up as multiple providers so they can add more locations (such as [West London Teaching School Alliance](https://web.archive.org/web/20210125160621/https://www.find-postgraduate-teacher-training.service.gov.uk/results/filter/provider?l=3&prev_l=none&prev_lat=none&prev_lng=none&prev_loc=none&prev_lq=none&prev_query=none&prev_rad=none&query=West+London+Teaching+School+Alliance)). Others only list one address (this is the case for 50% of SCITTs and 100% of HEIs).
```

Comment:
Do the acronyms SCHITT and HEI need acronym hover text explaining the full names?

Reply:
Good idea. We should also make it possible to link to individual definitions.